apps/chat/src/components/Common/Checkbox.tsx

```diff
@@ -0,0 +1,56 @@
+import { IconCheck, IconMinus } from '@tabler/icons-react';
+import { RefObject } from 'react';
+
+import classNames from 'classnames';
+
+interface CheckboxProps {
+  className?: string | boolean;
+  isSelected: boolean | undefined;
+  isPartialSelected?: boolean;
+  ref?: RefObject<HTMLInputElement>;
+  onChange:
+    | (() => void)
+    | ((event: React.ChangeEvent<HTMLInputElement>) => void);
+}
+export function Checkbox({
+  className,
+  isSelected,
+  isPartialSelected,
```

Comment:
Maybe better?
```suggestion
  checked,
  isPartialChecked,
```
because we are talking about checkbox (independent for which goals)

Reply:
Changed as suggested